$MirOS: ports/misc/mc/patches/patch-src_setup_h,v 1.2 2006/02/20 20:23:23 tg Exp $
--- src/setup.h.orig	Sun Aug 29 16:55:54 2004
+++ src/setup.h	Mon Feb 20 18:32:46 2006
@@ -25,7 +25,7 @@ extern int startup_left_mode;
 extern int startup_right_mode;
 extern int verbose;
 
-#define PROFILE_NAME     ".mc/ini"
-#define HOTLIST_FILENAME ".mc/hotlist"
+#define PROFILE_NAME     ".etc/mc/ini"
+#define HOTLIST_FILENAME ".etc/mc/hotlist"
 
 #endif				/* !__SETUP_H */
